From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <nornagon@nornagon.net>
Date: Mon, 13 Apr 2020 14:20:17 -0700
Subject: use --harmony-weak-refs-with-cleanup-some in test

Ref https://chromium-review.googlesource.com/c/v8/v8/+/2141011

cleanupSome is disabled with just `--harmony-weak-refs` due to a spec
disagreement.

diff --git a/test/parallel/test-finalization-group-error.js b/test/parallel/test-finalization-group-error.js
index 46a670073b1dbba9729e54166378991a7edba5a0..0857bedd043f0436bddc6d8641c51e78a8b4c562 100644
--- a/test/parallel/test-finalization-group-error.js
+++ b/test/parallel/test-finalization-group-error.js
@@ -1,6 +1,6 @@
 'use strict';
 
-// Flags: --expose-gc --harmony-weak-refs
+// Flags: --expose-gc --harmony-weak-refs-with-cleanup-some
 
 const common = require('../common');
 const assert = require('assert');
